shell/packages/sandstorm-accounts-ui/login_buttons.js

```diff
@@ -2,6 +2,9 @@
 var loginButtonsSession = Accounts._loginButtonsSession;
 
 var helpers = {
+  currentRoute: function (routeName) {
```

Comment:
I like that idea but I did copy-paste this thing from elsewhere.

Two options:

1. Keep it this name, to be consistent with the other route, or

2. Adjust the name here *and* everywhere

3. Adjust the name here, but leave everywhere else in its current
non-idiomaticness



I'll go for option 2. I don't mind a little refactoring while making
changes.